src/components/DBT/DBTTransformType.tsx

```diff
@@ -199,10 +203,51 @@ const DBTTransformType = ({
     }
   };
 
+  const fetchMoreLogs = async(flow_run_id: string, updateLimit: boolean) => {
+    let newMaxLimit = maxLogs;
+    if(updateLimit){
+      newMaxLimit += flowRunLogsOffsetLimit;
+    }
+    setMaxLogs(newMaxLimit);
+    await fetchLogs(flow_run_id, newMaxLimit);
+  }
+
+  const fetchLogs = async (flow_run_id = flowRunId, maxLogsLimit = maxLogs) => {
+    if (!flow_run_id) {
+      return;
+    }
+    setExpandLogs(true);
+    (async () => {
+      try {
+        const currLogsCount = dbtSetupLogsRef?.current?.length;
+        if (currLogsCount >= maxLogsLimit) {
+          return;
+        }
+        const data = await httpGet(
+          session,
+          `prefect/flow_runs/${flow_run_id}/logs?offset=${currLogsCount}&limit=${maxLogsLimit - currLogsCount}`
+        );
+
```

Comment:
The backend api `flow_runs/{flow_run_id}/logs` has a parameter of `task_run_id` also which we haven't used. 

this logic is a bit off, we first need to first graph of the `flow_run_id`. Then loop over the each task with `task_run_id` & display their logs with pagination. 

check the comment on the backend PR.